tests/unit/test_io_convert.py

```diff
@@ -163,6 +169,22 @@ def test_export_roundtrip(self):
                     container=container,
                     write_path=write_path,
                     export_path=export_path)
+                if self.REFERENCES:
+                    if self.TARGET_FORMAT == "H5":
+                        num_bazs = 10
+                        for i in range(num_bazs):
+                            baz_name = 'baz%d' % i
+                            self.assertEqual(exported_container.baz_data.data.__class__.__name__,
+                                             'ContainerH5ReferenceDataset')
```

Comment:
Can we evaluate the classes are equal instead of the class name? That's more typical. This formulation makes it look like there is a reason we cannot import these classes. e.g.,
`self.assertIsInstance(exported_container.baz_data.data, ContainerH5ReferenceDataset)`